homepages/homepage.php

```diff
@@ -5,23 +5,25 @@
  */
 function largo_register_default_homepage_layouts() {
 	// Load layouts from `layouts/`
-	$layouts = glob(__DIR__ . '/layouts/*.php');
-	foreach ($layouts as $layout)
-		include_once $layout;
-
-	// Load zone components from `zones/`
-	$zones = glob(__DIR__ . '/zones/*.php');
-	foreach ($zones as $zone)
-		include_once $zone;
-
 	$default_layouts = array(
 		'HomepageBlog',
 		'HomepageSingle',
 		'HomepageSingleWithFeatured',
 		'HomepageSingleWithSeriesStories',
 		'TopStories',
-		'LegacyThreeColumn'
+		'LegacyThreeColumn',
 	);
+	foreach( $default_layouts as $layout ) {
+		$layout_file = get_stylesheet_directory() . '/homepages/layouts/' . $layout . '.php';
```

Comment:
This should be `get_template_directory()` rather than `get_stylesheet_directory()`; in the case of a child theme this change would try to require these files from the child theme's directory.

Reply:
Of course. Fixed.